extensions/scheduler/runtime/src/test/java/io/quarkus/scheduler/runtime/util/SchedulerUtilsTest.java

```diff
@@ -3,6 +3,8 @@
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
+import io.quarkus.scheduler.common.runtime.util.SchedulerUtils;
```

Comment:
This test should go in the `common` module.

Reply:
Good catch, fixed